### PR TITLE
Initial draft of OPI lab spending guidelines doc

### PR DESCRIPTION
**Pull request by kkongusa**
Fixed "signed-off-by" from "Keith Kong" to "Kong, Keith".  Hope this will go through CI.

### CODE REVIEW

File: lab/spending_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+### Server peripherals
+
+- Hard drives
+- RAM
+- NIC cards
+
+### Test equipment
+
+- IXIA traffic generators
```
Comment (pdp2shirts) [inline lab/spending_guidelines.md:26]: Should this be genericized to simply "traffic generators"? i.e. not specifying one brand/product line of traffic generators?

File: lab/spending_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# OPI Lab Budget Guidelines
+
+This document provides guidance for using OPI funds to acquire lab
+equipment.  We describe both "appropriate" and "inappropriate" use of
+equipment funds and include examples of each.  Generally, commonly
```
Comment (pdp2shirts) [inline lab/spending_guidelines.md:5]: To be clear your intention is that the below is a set of examples not an exhaustive list of the appropriate equipment?

Comment (kkongusa) [reply lab/spending_guidelines.md:5]: Sounds reasonable to me.

File: lab/spending_guidelines.md (hunk 1 above)
Comment (kkongusa) [reply lab/spending_guidelines.md:26]: Sounds reasonable to me.

File: lab/spending_guidelines.md (hunk 2 above)
Comment (kkongusa) [reply lab/spending_guidelines.md:5]: Yes.  Two parts here.  First, that is why file name has "guideline".  Secondly, expectation is folks should issue additional PRs to augment this list.  I expect some neutral "equipment czar" to be designated by the TSC to have final say on purchase requests.  That person should exercise judgment to approve at least large purchase requests.

**Comment by seroyer** [comment]
@kkongusa every single commit you push must have signoff.  You can correct it without abandoning this PR.  Just do `git commit -s --amend` and `git push -f` to force replace your last commit (which is the one that's violating the DCO rules.

**Comment by kkongusa** [comment]
> @kkongusa every single commit you push must have signoff. You can correct it without abandoning this PR. Just do `git commit -s --amend` and `git push -f` to force replace your last commit (which is the one that's violating the DCO rules.

I guess the problem is there were two commits that did not have the signature.  I had to do this git rebase -i HEAD~2, use "reword" thingie, and then git --force push.

**Comment by kkongusa** [comment]
> @kkongusa it appears that this PR still needs DCO fixed before it can be merged.

I am going to go ahead and close this PR, since the file itself is already merged into its final resting spot at: https://github.com/opiproject/opi/blob/main/Policies/spending_guidelines.md  If folks still want to have the file stored in opi-poc, I will raise another PR.

**Pull request closed without merging**